src/docfx/config/DocumentIdConfig.cs

```diff
@@ -1,10 +1,17 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.Collections.Generic;
+using System.IO;
+
 namespace Microsoft.Docs.Build
 {
     internal class DocumentIdConfig
     {
+        private readonly Lazy<Dictionary<string, string>> _reversedDepotMappings;
+        private readonly Lazy<Dictionary<string, string>> _reversedDirectoryMappings;
```

Comment:
nit: put these computed values to `Docset` to align with other computed properties like _Redirections_, _RestoreMap_

Reply:
I followed the RouteConfig which only want to provide two methods for document id config.

---
In reply to: [203581997](https://github.com/dotnet/docfx/pull/3095#discussion_r203581997) [](ancestors = 203581997)